spring-cloud-stream-module-launcher/src/main/java/org/springframework/cloud/stream/module/resolver/ModuleResolverProperties.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.module.resolver;
+
+import java.io.File;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Configuration properties for {@link ModuleResolver}.
+ *
+ * @author Ilayaperumal Gopinathan
+ */
+@ConfigurationProperties
+public class ModuleResolverProperties {
+
+	/**
+	 * Local maven repository to resolve the module location.
+	 */
+	private File localRepository;
+
+	/**
+	 * Remote maven repository to resolve the module location.
+	 */
+	private String remoteRepository;
```

Comment:
+1 and that's being addressed here: https://github.com/spring-cloud/spring-cloud-stream/pull/83/files#diff-d0fe905e9297c6c526220607041efc5aR41